__tests__/index.test.js

```diff
@@ -202,6 +202,69 @@ describe("POST /api/articles/:article_id/comments", () => {
   });
 });
 
+describe.only("PATCH /api/articles/:article_id", () => {
+  test("PATCH: 200 should increment the specified article vote with the inc_votes provided", () => {
+    const newVotes = { inc_votes: 5 };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then(({ body }) => {
+        const { updatedArticle } = body;
+
+        expect(updatedArticle.votes).toBe(105);
+      });
+  });
+
+  test("PATCH: 200 should decrement the specified article vote with the inc_votes provided", () => {
+    const newVotes = { inc_votes: -5 };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then(({ body }) => {
+        const { updatedArticle } = body;
+        expect(updatedArticle.votes).toBe(95);
+      });
+  });
+
+  test("PATCH: 400 responds with an appropriate error when newVotes is not provided", () => {
+    const newVotes = {};
+
+    return request(app)
+      .patch("/api/articles/NaN")
+      .send(newVotes)
```

Comment:
You have 2 errors here. The empty req.body and the NaN. An error test should only test one thing, otherwise we cant be sure they are both being handled.

Reply:
Just change Nan to a real number and you should be fine :)